doc/code/qml_templates.rst

```diff
@@ -7,6 +7,12 @@ evaluate, and train quantum nodes.
 
 .. currentmodule:: pennylane.templates
 
+Base Templates
+--------------
```

Comment:
Are these base templates, or more 'template constructors'(?)

Reply:
I am not so sure any more...to me they are conceptually not different to any other templates in the way they are used...